db/seeds.rb

```
@@ -6,5 +6,9 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 (20..39).each do |id|
-  Image.create(url: "https://picsum.photos/id/#{id}/400/400.jpg")
+  image = Image.create(url: "https://picsum.photos/id/#{id}/400/400.jpg")
+  %w[foo bar baz plugh xyzzy fred qux bazz foobar quxx].sample(3).each do |tag|
+    image.tag_list.add(tag)
+  end
+  image.save
```

for this I would suggest using `image.save!` so this raises an exception if it fails

---

Ah, thank you - I was pretty annoyed by silent failures 👍